<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Lenovo-PC" failures="1" tests="10" name="Test" time="0.146" errors="0" timestamp="2023-07-23T20:36:20 IST">
  <testcase classname="testng.test.FirstTest" name="user1SignIn" time="0.012"/>
  <testcase classname="testng.test.FirstTest" name="user1Login" time="0.002"/>
  <testcase classname="testng.test.SecondTest" name="user2Login" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.test.SecondTest.user2Login(SecondTest.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- user2Login -->
  <testcase classname="testng.test.SecondTest" name="user2SignIn" time="0.002"/>
  <testcase classname="testng.test.ThirdTest" name="safeLogin" time="0.002"/>
  <testcase classname="testng.test.ThirdTest" name="user3Login" time="0.002"/>
  <testcase classname="testng.test.ThirdTest" name="user3SignIn" time="0.001"/>
  <testcase classname="testng.test.FourthTest" name="SignInForm" time="0.002"/>
  <testcase classname="testng.test.FourthTest" name="SignInForm" time="0.002"/>
  <testcase classname="testng.test.FourthTest" name="SignInForm" time="0.002"/>
</testsuite> <!-- Test -->
